wise/type_ast.flow

```diff
@@ -0,0 +1,88 @@
+import wise/wise_types;
+import ds/tree;
+
+export {
+	typeWiseAst(ast : WiseAst) -> string;
```

Comment:
We have to introduce type classes and do the unification algorithm to get this right.